perf_test/sparse/KokkosSparse_run_spadd.hpp

```diff
@@ -0,0 +1,159 @@
+/*
+//@HEADER
+// ************************************************************************
+//
+//               KokkosKernels 0.9: Linear Algebra and Graph Kernels
+//                 Copyright 2017 Sandia Corporation
+//
+// Under the terms of Contract DE-AC04-94AL85000 with Sandia Corporation,
+// the U.S. Government retains certain rights in this software.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are
+// met:
+//
+// 1. Redistributions of source code must retain the above copyright
+// notice, this list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright
+// notice, this list of conditions and the following disclaimer in the
+// documentation and/or other materials provided with the distribution.
+//
+// 3. Neither the name of the Corporation nor the names of the
+// contributors may be used to endorse or promote products derived from
+// this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY SANDIA CORPORATION "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL SANDIA CORPORATION OR THE
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+// LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+// NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// Questions? Contact Siva Rajamanickam (srajama@sandia.gov)
+//
+// ************************************************************************
+//@HEADER
+*/
+
+#include "KokkosKernels_Handle.hpp"
+#include "KokkosKernels_IOUtils.hpp"
+#include "KokkosSparse_spadd.hpp"
+#include "KokkosKernels_TestParameters.hpp"
+
+namespace KokkosKernels {
+namespace Experiment {
+
+template <typename crsMat_t>
+void run_experiment(Parameters params)
+{
+  using namespace KokkosSparse;
+  using namespace KokkosSparse::Experimental;
+
+  using size_type = typename crsMat_t::size_type;
+  using lno_t = typename crsMat_t::ordinal_type;
+  using scalar_t = typename crsMat_t::value_type;
+  using device_t = typename crsMat_t::device_type;
+  using exec_space = typename device_t::execution_space;
+  using mem_space = typename device_t::memory_space;
+
+  using KernelHandle = KokkosKernels::Experimental::KokkosKernelsHandle
```

Comment:
Are these things still in the Experimental namespace? or have they been imported into KokkosKernels?

Reply:
@mhoemmen Yes, all the sparse kernels are still in this namespace.

Reply:
@brian-kelley So is @trilinos/muelu depending nonoptionally on experimental code?

Reply:
@mhoemmen Technically yes, and Tpetra as well (spgemm/spadd). But really, these things are all several years old and have stable interfaces, so we should talk about removing the Experimental namespace.